pytorch_lightning/loggers/tensorboard.py

```diff
@@ -17,6 +17,13 @@
 from pytorch_lightning.loggers.base import LightningLoggerBase, rank_zero_experiment
 from pytorch_lightning.utilities import rank_zero_only
 
+try:
+    from omegaconf import Container, OmegaConf
+except ImportError:
+    OMEGACONF_AVAILABLE = False
+else:
+    OMEGACONF_AVAILABLE = True
+
```

Comment:
just wanted to point out (nitpick) that importerror is not the right exception to catch. What you try to do here is check if the package is available, which can be done without trying to import it with just one line with `importlib`. If the package is available, there can still be an importerror for a different reason. In such a case we really WANT to see the stacktrace and not just ignore it. See also #2266 

This actually occurs in many places of PL. Would be great to clean up these code smells and eliminate them right away in the review process of new PRs @PyTorchLightning/core-contributors